src/steps/2.addModules/moduleConfigs.ts

```diff
@@ -449,12 +482,15 @@ export const moduleConfigs: Record<Modules, ModuleConfig> = {
       },
     ],
     tasksPostInstall: [],
-    htmlForIndexVue: `<p>
-      Checkout the tRPC demo page here:
-      <nuxt-link to="/trpc" class="underline text-blue">
-        Click me to test the tRPC setup!
-      </nuxt-link>
-    </p>`
+    indexVue: {
+      ...generateModuleHTMLSnippet(
+        "tRPC",
+        "tRPC allows you to easily build & consume fully typesafe APIs without schemas or code generation.",
+        "trpc__card",
+        "https://sidebase.io/sidebase/components/trpc",
+        "background: radial-gradient(#f588d8, #c0a3e5);"
+      ),
+    },
```

Comment:
no need to spread here and in basically all other places:
```suggestion
    indexVue: generateModuleHTMLSnippet(
        "tRPC",
        "tRPC allows you to easily build & consume fully typesafe APIs without schemas or code generation.",
        "trpc__card",
        "https://sidebase.io/sidebase/components/trpc",
        "background: radial-gradient(#f588d8, #c0a3e5);"
    ),
```